nodes/solid/solid_distance.py

```diff
@@ -0,0 +1,164 @@
+
+from sverchok.dependencies import FreeCAD
+from sverchok.utils.dummy_nodes import add_dummy
+
+if FreeCAD is None:
+    add_dummy('SvSolidDistanceNode', 'Solid Distance', 'FreeCAD')
+else:
+    import bpy
```

Comment:
What is the reason of coding all node in the else condition? Extra nestedness is not good.

Reply:
Is not my favourite either but is the convention I took from sverchok-extra and that now is in all the nodes with dependencies

Reply:
@Durman the original problem is that we do not want use "FreeCAD" symbol (in this case) to be used in case it is None. It would be good if we could write something like
```python
if FreeCAD is None:
    add_dummy(...)
    return_from_module

class SvSomeNode...
```
but python does not have such construct afaik...

Reply:
@portnov Did not understand. If the class will be declared outside else statement what it effects?